README.md

```diff
@@ -1,243 +1,59 @@
-
-### Grading Criteria
-You will be graded as follows:
-
-|| Meets expectations (9-10) | Some Issues (6-8) | Does not meet expectations (0-5)|
-|---|---|---|---|
-|Coverage| Relevant issues are created in the issue tracker. | Issues do not capture initial tasking. | Few or no issues are present. |
-|Labeling and assignment| Issues use GitHub tags and are assigned to a team member | Some issues are labeled and assigned. | Few or no labels and assignments are used.|
-
-**20 points**
-
-## Presentation
-You will be expected to present your Milestone 1 proposal to the class. It is important that you use this time to inform your classmates and get feedback. Things to be considered are 1) conveying a sense of interest and excitement about your project, 2) soliciting feedback from peers that might be useful for development or refinement of the project, and 3) presenting your methodology and scope to get recommendations from peers and the instructor.
-
-### Submission Materials
-You will record your presentation as a video using VidGrid. Post the video to the canvas submission page AND in the slack `#sp-2021-general` chat. Also submit your slides on canvas.
-
-### Grading Criteria
-You will be graded by a presentation rubric on canvas.
-
-**Total 70 points.**
-
-## Teamwork
-Lastly, it is important that you realize you are working on a TEAM (except for the few of you that are soloing). I will say now, that your grade is dependent on your participation. I will not allow individuals to sluff off on a team. Your grade is therefore based partly on a *participation factor*. Essentially if you don't participate (and trust me We will know) - you will not get all of the team points on your project grade.
-
-Your final grade on the milestone is computed as follows:
-**team_grade** * **participation_factor** = **individual_grade**
-
-**participation_factor** is a number from 0 to 1 that is based on four measured participation factors:
-
-1. Contribution to the GitHub repo
-1. Activity in the slack channel for your project
-1. Activity on GitHub Projects
-1. In-class participation and participation in team meetings.
-1. Team evaluations
-
-If you do your work and participate, you will get the full team points (**participation_factor**=1), if you don't do anything, you will get 0% of the team points, if you aren't doing enough you will suffer a correspondingly appropriate penalty to your grade. This means that you can **FAIL** even if your team succeeds - so don't sluff off and expect the team to carry you.
+|Some resource| No | Bob | Some description  |
+|e.g. PLC unit | Yes | Gary | A programmable logic controller from Siemens for investigation.|
```

Comment:
Don't we need to list the Gantt tool at here? I just wondered

Reply:
I agree! This was just the placeholder from Dr. Hale's version which I copied purely for the formatting though, so I'll leave that for Bryan to add when he does that section.